image/postgres/Dockerfile.envsubst

```
@@ -19,6 +19,7 @@ ENV ROX_IMAGE_FLAVOR=${ROX_IMAGE_FLAVOR}
 
 ENV PG_MAJOR=13
 ENV PATH="$PATH:/usr/pgsql-$PG_MAJOR/bin/"
+ENV LANG en_US.utf8
```

This should be
```suggestion
ENV LANG=en_US.utf8
```
right?